tinygrad/tensor.py

```diff
@@ -204,7 +204,7 @@ def leakyrelu(self, neg_slope=0.01):
     return self.relu() + (-neg_slope*self).relu()
 
   def abs(self):
-    return self.relu() + (-1.0*self).relu()*(-1.0)
+    return self.relu() + (-1.0*self).relu()
```

Comment:
def leakyrelu(self, neg_slope=0.01):
    return self.relu() + (-neg_slope*self).relu()

isnt the above leakyrelu with neg_slope = -1.0

Reply:
why not use it as 
def abs(self):
   return self.leakyrelu(1.0)